tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build",
+    "outDir": "/genfiles",
```

Comment:
What is the slash relative to here, it looks like an absolute path? It feels like this should be just `genfiles` or `./genfiles`.

Reply:
I was trying to see if it fixed the tsickle error, but it doesn't. Left it as "genfiles"